examples/ui/gradio/gradio-hftgi-rag-redis/app.py

```diff
@@ -5,7 +5,7 @@
 from queue import Empty, Queue
 from threading import Thread
 from typing import Optional
-
+from text_generation import Client
```

Comment:
@tsisodia10 will this fail if we are using [requirements](https://github.com/RHEcosystemAppEng/llm-on-openshift/blob/main/examples/ui/gradio/gradio-hftgi-rag-redis/Dockerfile#L10) to load [libraries](https://github.com/RHEcosystemAppEng/llm-on-openshift/blob/main/examples/ui/gradio/gradio-hftgi-rag-redis/requirements.txt#L82)?

Reply:
@tsisodia10 looks like this is resolved!